README.md

```diff
@@ -49,19 +49,20 @@ brew install llvm
 
 After installing with Homebrew, you may need to set the following CMake arguments:
 
-```{sh}
--DCMAKE_CXX_COMPILER=/Users/username/.local/homebrew/opt/llvm/bin/clang++
--DOMP_ROOT=/Users/username/.local/homebrew/opt/llvm/
--DCXX_ROOT=/Users/username/.local/homebrew/opt/llvm
-```
+    ```{sh}
+    -DCMAKE_CXX_COMPILER=/Users/username/.local/homebrew/opt/llvm/bin/clang++
+    -DOMP_ROOT=/Users/username/.local/homebrew/opt/llvm/
+    -DCXX_ROOT=/Users/username/.local/homebrew/opt/llvm
+    -DHDF5_ROOT=/Users/username/.local/homebrew/opt/hdf5
+    ```
```

Comment:
Revert the weird indentation
`````suggestion
```{sh}
-DCMAKE_CXX_COMPILER=/Users/username/.local/homebrew/opt/llvm/bin/clang++
-DOMP_ROOT=/Users/username/.local/homebrew/opt/llvm/
-DCXX_ROOT=/Users/username/.local/homebrew/opt/llvm
-DHDF5_ROOT=/Users/username/.local/homebrew/opt/hdf5
```
`````